listenbrainz/webserver/static/js/jsx/spotify-player.jsx

```diff
@@ -339,6 +348,7 @@ export class SpotifyPlayer extends React.Component {
         >
           {this.getAlbumArt()}
         </PlaybackControls>
+        <div className='btn' onClick={this.disconnectSpotifyPlayer}>Sneaky disconnect for testing</div>
```

Comment:
LOL

Reply:
Yeah. Maybe I could do something a bit more subtle.

Reply:
Naw, fine for now. :)

Reply:
I attached the disconnectSpotifyPlayer to the window object, so you can call `disconnectSpotifyPlayer()` straight from the browser console. Slightly less ugly.